docs/project_overview.md

```diff
@@ -113,20 +113,13 @@ ecosystem. Access to this file is handled through the
 [`DAO`](https://github.com/qiskit-community/ecosystem/blob/main/ecosystem/daos/dao.py)
```

Comment:
Outdated - this whole file probably needs a read-through, like line 102

Reply:
Yeah for accessing the TOML files programmatically (e.g. update stars, build the website).